### PR TITLE
Support custom progress position

### PR DESCRIPTION
**Pull request by zzzworm**
I add custom progress position support for my project. and fixed some miss use of UIVisualEffectView for addSubView

### CODE REVIEW

File: LNPopupController/LNPopupController/LNPopupBar.h [hunk 1]
```diff
@@ -39,6 +39,19 @@ typedef NS_ENUM(NSUInteger, LNPopupBarStyle) {
 	LNPopupBarStyleCustom
 };
 
+typedef NS_ENUM(NSUInteger, LNPopupBarProgressStyle) {
```
Comment (LeoNatan) [inline LNPopupController/LNPopupController/LNPopupBar.h]: Let’s call it `LNPopupBarProgressViewPosition`.

File: LNPopupController/LNPopupController/LNPopupBar.h [hunk 2]
```diff
@@ -89,6 +102,11 @@ typedef NS_ENUM(NSUInteger, LNPopupBarStyle) {
 @property (nonatomic, assign) LNPopupBarStyle barStyle UI_APPEARANCE_SELECTOR;
 
 /**
+ *  The popup bar's progress style.
+ */
+@property (nonatomic, assign) LNPopupBarProgressStyle progressStyle;
```
Comment (LeoNatan) [inline LNPopupController/LNPopupController/LNPopupBar.h]: `progressViewPosition`

File: LNPopupController/LNPopupController/Private/LNPopupBar.m [hunk 3]
```diff
@@ -174,13 +176,15 @@ - (nonnull instancetype)initWithFrame:(CGRect)frame
...
-		[_backgroundView addSubview:_titlesView];
+		[_backgroundView.contentView addSubview:_titlesView];
 		
 		_progressView = [[UIProgressView alloc] initWithProgressViewStyle:UIProgressViewStyleDefault];
 		_progressView.translatesAutoresizingMaskIntoConstraints = NO;
 		_progressView.trackImage = [UIImage alloc];
 		[self addSubview:_progressView];
-		[self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:[_progressView(2)]|" options:0 metrics:nil views:NSDictionaryOfVariableBindings(_progressView)]];
+        
+        [self _updateProgressStyle];
```
Comment (LeoNatan) [inline LNPopupController/LNPopupController/Private/LNPopupBar.m]: Please convert spaces to tabs, as per other whitespace in the project. Thanks

File: LNPopupController/LNPopupController/Private/LNPopupBar.m [hunk 4]
```diff
@@ -347,6 +363,17 @@ - (void)setSystemBarStyle:(UIBarStyle)systemBarStyle
 	[self _innerSetBackgroundStyle:_userBackgroundStyle];
 }
 
+- (void)setProgressStyle:(LNPopupBarProgressStyle)progressStyle
+{
+    if (_progressStyle != progressStyle) {
```
Comment (LeoNatan) [inline LNPopupController/LNPopupController/Private/LNPopupBar.m]: Let’s please have the opening `{` brackets at the start of a new line. Thanks

File: LNPopupControllerExample/LNPopupControllerExample/FirstViewController.m [hunk 5]
```diff
@@ -122,7 +122,7 @@ - (IBAction)_presentBar:(id)sender
 	targetVC.popupContentView.popupCloseButton.accessibilityHint = NSLocalizedString(@"Custom popup button accessibility hint", @"");
 	
 	targetVC.popupBar.previewingDelegate = self;
-	
+    targetVC.popupBar.progressStyle = LNPopupBarProgressStyleTop;
```
Comment (LeoNatan) [inline LNPopupControllerExample/LNPopupControllerExample/FirstViewController.m]: This breaks the build. It should also be updated to the new naming convention. Thanks

Comment (zzzworm) [inline LNPopupControllerExample/LNPopupControllerExample/FirstViewController.m]: Sorry, I did't check it.I will commit a new request

**Comment by LeoNatan** [comment]
Approved! Thank you for your request and willingness to take feedback!

**Pull request merged**